src/main/java/carpet/script/value/NumericValue.java

```diff
@@ -201,11 +201,11 @@ public boolean equals(Object o)
 
     public NumericValue(double value)
     {
-        this.value = value;
+        this(value, (long) value);
```

Comment:
why?

Reply:
So then I can avoid repeating the if-else statement over and over ig? if u want I can remove ofc

Reply:
actually nah Imma just remove it and add the if else thing there to. Literally the same thing so who cares